pypesto/history/hdf5.py

```diff
@@ -362,7 +419,7 @@ def get_time_trace(
         """See `HistoryBase` docstring."""
         return self._get_hdf5_entries(TIME, ix)
 
-    def _is_editable(self, file: str) -> bool:
+    def _editable(self) -> bool:
```

Comment:
Why no `with_h5_file` here?

Reply:
because this function is evaluated at the beginning, and thus has an additional catch to just flag as not editable, but not raise an error.